apps/mobile/App.tsx

```diff
@@ -54,13 +53,11 @@ function AppShell({children}: React.PropsWithChildren) {
       <ConnectionProvider>
         <ThemeProvider storage={themeStorageKeyManager}>
           <LanguageProvider storage={languageStorageKeyManager}>
-            <QueryProvider>
-              <CopyProvider>
-                <CrashBoundary>
-                  <LoadingOverlayProvider>{children}</LoadingOverlayProvider>
-                </CrashBoundary>
-              </CopyProvider>
-            </QueryProvider>
+            <CopyProvider>
```

Comment:
conflits with `ConnectionProvider.tsx` beacuse it has `PersistQueryClientProvider`.